changelog/1.16.0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "1.16.0"
+description: "Released on 02/22/2021"
+---
+
+### Breaking Changes ❗
+
+- The deprecated `Services` feature is now **removed**. All resources created
+  for Services are preserved for data recovery purposes (volumes,
+  database tables, etc.), but are not accessible through the Coder platform.
+  Please reference
+  [Docker in Environments](../environments/cvms) for
+  a guide on the new workflow for running containers with your Environment.
+- The built-in Internal Extension Marketplace is now **removed**. All resources
+  created for this feature are preserved for data recover purposes, but are not
+  accessible through the Coder platform. For air-gapped deployments, the `Custom`
+  Extension Marketplace configuration enables the use of
+  [Open VSX](https://github.com/eclipse/openvsx) as
+  an alternative. For more details, please view our [migration guide](TODO).
+- The administrator configuration option "Privileged Environments" is now
+  removed. All non-CVM environments are launched with `privileged: false`.
+- The [Container-based Virtual Machine](../admin/environment-management/cvms)
+  environment option is now considered generally available. The
+  admin-level configuration is still required, but the `alpha`
+  warning has been removed.
+- Environment SSH routing has new behavior. Please reference
```

Comment:
Is this breaking? I'm not familiar with what changed, so if it's breaking the actual breaking part should be described here.

Reply:
It is for environment variable usage via SSH without configuration on the users end. That might be worth mentioning.

Reply:
If that's the case it doesn't seem to be breaking then and this should be moved to features.

Reply:
It _is_ a breaking change in cases where the user was intentionally or unintentionally running a TCP server on port 22 inside their environment.

Reply:
The new behavior is described in greater detail at the link provided.

Reply:
This should read:
> Environment SSH routing has new behavior. If you have any processes listening on port 22 it will be used to provide SSH access to your environment instead of the bundled SSH implementation. This is a breaking change for users that had services listening on port 22 before this change. Please reference blah blah

Reply:
When we describe breaking changes we should explain what broke and who it affects in a clear to understand way without linking around to other resources. It should be obvious from the changelog who this change affects and what steps they need to take to mitigate the impact.

Reply:
Wording looks great, we should add that